codelabs/codelab-03/README.md

```diff
@@ -0,0 +1,143 @@
+# Codelab 3: AWS + S3 + ClounFront
+
+### Due Date
+
+This codelab comes in two parts, both of which are due on *Thursday, February 15th at 11:59:59PM*. This is codelab-03 (part B) which expands upon what was learned in [codelab-02](https://umd-cs-stics.gitbooks.io/cmsc389l-practical-cloud-computing-spring-2018/content/codelabs/codelab-02/) (part A) and introduces CloudFront.
+
+### Goal
+
+In this codelab, you'll get to play around with ClounFront.
+- You'll set up CloudFront via the AWS GUI
+- You'll test latency 
+- You'll issue an invalidation
+
+### Setting Up
+
+Before starting this codelab, run `git pull` in the `389Lspring18` directory to update your local copy of the class repository.
+
+### Tutorial
+
+**It's a good idea to read/skim the entire codelab first so you have an idea of what you're doing. At a minimum, look at the submission section to _avoid having to redo your work_ to obtain screenshots**
```

Comment:
maybe rephrase as:

**Keep in mind that you will be submitting screenshots of some command outputs, so make sure to read the submission section at the bottom to understand what you will be turning in.**